ligo/haskell/test/Test/Ligo/BaseDAO/Proposal.hs

```diff
@@ -7,10 +7,62 @@ module Test.Ligo.BaseDAO.Proposal
 
```

Comment:
It looks like your tests are now only failing due to `Natural` not being in scope, it suggests you to take it from `Lorentz`, but you can also just use `Universum`:
```suggestion

import Universum

```
and it should be fine.